docker-run.sh

```diff
@@ -2,8 +2,19 @@
 
 set -e
 
-INDEX_FILE=/usr/share/nginx/html/index.html
+NGINX_ROOT=/usr/share/nginx/html
+INDEX_FILE=$NGINX_ROOT/index.html
 
 # TODO: this is empty but we'll be adding configuration values here
 
-exec nginx -g 'daemon off;'
\ No newline at end of file
+## Adding env var support for swagger json
+if [[ -f $SWAGGER_JSON ]]; then
```

Comment:
I think we can use `SWAGGER_FILE` or `DEFAULT_FILE` instead since it's working with both `json` and `yaml`

Reply:
yes, makes sense

Reply:
Fixed in 675812c